report/main.qmd

```diff
@@ -0,0 +1,139 @@
+---
+title: "**STAT550 Homework No 2: Advice for Evaluating Interventions on Sugary and Zero-Calorie Beverage Consumption**"
+author: Son Luu (71843379), Xihan Qian (54285556) and Javier Mtz.-Rdz. (94785938)
+pdf-engine: pdflatex
+align: left
+date: "March 1, 2024"
+date-format: long
+number-sections: true
+number-depth: 4
+editor: visual
+format: 
+  pdf:
+    citeproc: false
+    filters: 
+      - at: pre-render
+        path: "../misc/citeproc.lua"
+      - at: pre-render
+        path: "../misc/wordcount.lua"
+    documentclass: article
+    header-includes: |
+      \usepackage[left=0.8in,right=0.8in,
+      top=0.8in,bottom=1in,footskip=0.5in]{geometry} 
+      \usepackage[document]{ragged2e}
+      \usepackage{amsmath,amsthm,amssymb,amsfonts}
+      \usepackage{mathtools}
+      \usepackage{dsfont}
+      \usepackage{centernot}
+      \usepackage[usenames,dvipsnames,table]{xcolor}
+      \usepackage{booktabs} % For improved table lines
+      \renewcommand{\arraystretch}{1} % Increase row spacing
+      \renewcommand\thefigure{\arabic{figure}}
+    fontsize: 12pt
+    colorlinks: true
+bibliography: ../reference/ref.bib
+---
+
+```{=tex}
+\thispagestyle{empty}
+\clearpage
+\pagenumbering{arabic}
+```
+# Introduction
+
+Zero-calorie beverages offer an alternative to sugary drinks that can help to avoid the harmful effects of artificial sweeteners. Therefore, it is important to understand what actions can motivate people to switch to these products. This study aims to assess the impact of messaging and discounts on the buying behaviour of zero-calorie and sugary drinks. The study’s statistical advice outlines how to quantitatively evaluate four research questions on this topic. The first question to be addressed is whether these interventions lead to increased consumption of zero-calorie beverages and decreased consumption of sugary beverages. Subsequently, the study will also investigate whether the effects vary across different sites (second question) and whether combining interventions results in larger effects (third question). Lastly, the study will compare the effectiveness of calorie-equivalent messaging with simple calorie admonishment (fourth question).
```

Comment:
I think there are a lot of similarities between the first question and third/fourth question so maybe you can combine the three into one question.

Reply:
Same, and I would suggest to use \begin{enumerate} to make the structure more clear.